build.gradle

```diff
@@ -97,6 +97,12 @@ allprojects {
       dependency group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.4.2'
       dependency group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.13.4'
 
+
+      dependency group: 'org.slf4j', name: 'slf4j-api', version: '2.0.5'
+      dependency group: 'org.slf4j', name: 'slf4j-simple', version: '2.0.5'
+      dependency group: 'org.slf4j', name: 'jul-to-slf4j', version: '2.0.5'
+      dependency group: 'org.slf4j', name: 'slf4j-nop', version: '2.0.5'
+
```

Comment:
Put a comment above these four stating why we are overriding the imported version.
Rationale: we normally use this for CVEs but in this case it's not about the CVEs but about conflicting versions among other libraries, so we need to be aware

Reply:
Done